snippets/product-card.liquid

```diff
@@ -17,111 +17,109 @@
-    <span class="visually-hidden">{{ product_card_product.title | escape }}</span>
+  <div class="card-information">
+    <div class="card-information__wrapper">
+      {%- if show_vendor -%}
+        <span class="visually-hidden">{{ 'accessibility.vendor' | t }}</span>
+        <div class="caption-with-letter-spacing light">{{ product_card_product.vendor }}</div>
+      {%- endif -%}
 
-    <div class="card card--product{% if product_card_product.featured_media == nil %} card--text-only card--soft{% endif %}{% if product_card_product.featured_media != nil and show_image_outline %} card--outline{% endif %}" tabindex="-1">
-      <div class="card__inner">
-        {%- if product_card_product.featured_media -%}
-          {%- liquid
-            assign featured_media_aspect_ratio = product_card_product.featured_media.aspect_ratio
+      {%- if product_card_product.featured_media -%}
+        <h3 class="card-information__text h5">
+          <a href="{{ product_card_product.url | default: '#' }}" class="full-unstyled-link">
+            {{ product_card_product.title | escape }}
+          </a>
+        </h3>
+      {%- endif -%}
+
+      {% comment %} TODO: metafield {% endcomment %}
+      <span class="caption-large light">{{ block.settings.description | escape }}</span>
+      {%- if show_rating and product_card_product.metafields.reviews.rating.value != blank -%}
+        {% liquid
+          assign rating_decimal = 0 
+          assign decimal = product_card_product.metafields.reviews.rating.value.rating | modulo: 1 
+          if decimal >= 0.3 and decimal <= 0.7
+            assign rating_decimal = 0.5
+          elsif decimal > 0.7
+            assign rating_decimal = 1
+          endif 
+        %}
+        <div class="rating" role="img" aria-label="{{ 'accessibility.star_reviews_info' | t: rating_value: product_card_product.metafields.reviews.rating.value, rating_max: product_card_product.metafields.reviews.rating.value.scale_max }}">
+          <span aria-hidden="true" class="rating-star color-icon-{{ settings.accent_icons }}" style="--rating: {{ product_card_product.metafields.reviews.rating.value.rating | floor }}; --rating-max: {{ product_card_product.metafields.reviews.rating.value.scale_max }}; --rating-decimal: {{ rating_decimal }};"></span>
+        </div>
+        <p class="rating-text caption">
+          <span aria-hidden="true">{{ product_card_product.metafields.reviews.rating.value }} / {{ product_card_product.metafields.reviews.rating.value.scale_max }}</span>
+        </p>
+        <p class="rating-count caption">
+          <span aria-hidden="true">({{ product_card_product.metafields.reviews.rating_count }})</span>
+          <span class="visually-hidden">{{ product_card_product.metafields.reviews.rating_count }} {{ "accessibility.total_reviews" | t }}</span>
+        </p>
+      {%- endif -%}
+      {% render 'price', product: product_card_product, price_class: '' %}
```

Comment:
It seems to be working with me: https://screenshot.click/15-54-bzuec-48bvw.mp4

Reply:
Im just navigating using the `tab` key maybe I'm missing something to see the rest of the info: [video](https://screenshot.click/21-09-m0kjr-bz0r3.mp4)

Reply:
This is by design. Screen reader users navigate via virtual/browse mode by default. This is how non focusable content is consumed.

- VO: <kbd>Ctrl</kbd> + <kbd>Opt</kbd> + <kbd>→</kbd> (or <kbd>←</kbd> to navigate backward)
- NVDA: <kbd>↓</kbd> (or <kbd>↑</kbd> to navigate backward)
- JAWS: <kbd>↓</kbd> (or <kbd>↑</kbd> to navigate backward)

[More on screen reader interaction modes](https://tink.uk/understanding-screen-reader-interaction-modes/).

Reply:
Oh, from the video, you are using `tab` key to navigate all interactive (focusable) elements (a, input, button, details/summary...) so it's an expected behavior.

You use the screen reader's "virtual cursor" to read other elements that are not focusable (text, paragraph, heading...)

cc. @svinkle correct me if I'm wrong

So in this PR:
Before the change, if we `tab` key the product cards, it would read everything including the price.
After the change: if we `tab` key the product cards, it would only read the product's title.